modules/openapi-generator/src/main/resources/python/model.mustache

```diff
@@ -197,27 +197,37 @@ class {{classname}}(object):
         return self.discriminator_value_class_map.get(discriminator_value)
 
 {{/discriminator}}
-    def to_dict(self):
+    def to_dict(self, serialize=False):
         """Returns the model properties as a dict"""
         result = {}
 
+        def convert(x):
+            if hasattr(x, "to_dict"):
+                args = inspect.getargspec(x.to_dict).args
+                if len(args) == 1:
+                    return x.to_dict()
+                elif len(args) == 2:
+                    return x.to_dict(serialize)
+                else:
+                    raise ValueError("Invalid argument size of to_dict")
```

Comment:
Why would to_dict have more than 2 args?

Reply:
Hi @spacether , 
A model generated by newer openapi-generator-cli.jar should be able to apply to_dict function to its inner models which having only to_dict() but no to_dict(serialize) if those inner models are generated by an older openapi-generator-cli.jar
So I updated  to following code :

`        def convert(x):
            if hasattr(x, "to_dict"):
                args = inspect.getargspec(x.to_dict).args
                if len(args) == 1:
                    return x.to_dict()
                else:
                    return x.to_dict(serialize)
            else:
                return x
`

Reply:
Hi @spacether ,  is the change ok to be merged?

Reply:
Hi @spacether  , would you please help to review this new change?